test/LiquidityManager/invest.test.ts

```diff
@@ -123,12 +157,82 @@ describe.only('LiquidityManager#invest', () => {
         )
     }
 
+    function getRangeTicks(
+        pool: Pool,
+        lowerPricePercentage: number,
+        upperPricePercentage: number,
+    ) {
+        const currentPrice = pool.token0Price.asFraction
+
+        const lowerPrice = currentPrice.subtract(
+            currentPrice.divide(lowerPricePercentage),
+        ).toFixed(8)
+
+        const upperPrice = currentPrice.add(
+            currentPrice.divide(upperPricePercentage),
+        ).toFixed(8)
+
+        return {
+            tickLower: parsePriceToTick(pool.token0, pool.token1, pool.fee, lowerPrice),
+            tickUpper: parsePriceToTick(pool.token0, pool.token1, pool.fee, upperPrice),
+        }
+    }
+
+    function validateInvestTransaction(
+        amount0: bigint,
+        token0Price: BigNumber,
+        amount1: bigint,
+        token1Price: BigNumber,
+        receipt: ContractTransactionReceipt | null,
+    ) {
+        expect(receipt).to.not.be.undefined
```

Comment:
does null also fall under this expect?

Reply:
I'll change to check the instance of receipt